### PR TITLE
added till set 7

### PR DESCRIPTION
**Pull request by Abdullahjavednesar**
yet to add few tests

### Following functions needs to be implemented
- [ ] CalculusFreeQ
- [ ] CalculusQ
- [ ] HeldFormQ
- [x] SimplerSqrtQ
- [x] GeneralizedBinomialMatchQ

### Tests:
- [x] SumSimplerQ
- [x] SumSimplerAuxQ
- [x] SimplerIntegrandQ
- [x] CancelCommonFactors
- [x] TrinomialDegree
- [x] TrinomialMatchQ
- [x] QuadraticMatchQ
- [x] PowerOfLinearMatchQ
- [x] GeneralizedTrinomialMatchQ
- [x] QuotientOfLinearsMatchQ
- [x] PolynomialTermQ
- [x] PolynomialTerms
- [x] NonpolynomialTerms
- [x] PseudoBinomialQ
- [x] PseudoBinomialPairQ
- [x] NormalizePseudoBinomial
- [x] TrinomialParts

### CODE REVIEW

File: sympy/rubi/utility_function.py [hunk 1]
```diff
@@ -119,9 +123,12 @@ def NegativeOrZeroQ(u):
 def FractionOrNegativeQ(u):
     return FractionQ(u) or NegativeQ(u)
 
+<<<<<<< HEAD
```
Comment (arihantparsoya) [inline sympy/rubi/utility_function.py]: what is this line for?

File: sympy/rubi/utility_function.py [hunk 2]
```diff
@@ -1785,6 +1802,418 @@ def ExpandIntegrand(expr, x, extra=None):
...
+        return Drop(lst1, 2) == Drop(lst2, 2)
+
+def PseudoBinomialQ(u, x):
+    lst = PseudoBinomialParts(u, x)
+    if lst:
+        return True
+    else:
+        return False
+
+def Drop(lst, n):
```
Comment (parsoyaarihant) [inline sympy/rubi/utility_function.py]: add tests for drop too

File: sympy/rubi/tests/test_utility_function.py [hunk 3]
```diff
@@ -854,6 +854,120 @@ def test_NonnumericFactors():
 def test_Prepend():
     assert Prepend([1, 2, 3], [4, 5]) == [4, 5, 1, 2, 3]
 
+def test_CombineExponents():
```
Comment (arihantparsoya) [inline sympy/rubi/tests/test_utility_function.py]: its defined twice https://github.com/parsoyaarihant/sympy/pull/9/files/f0140167a205d4f372207103ffcce973e27d0217..836ab2cb8f76cc66e86e910b9b0035783e90d9f4#diff-186cc51357ab69cbb39b22a3a5254a8cR987

**Comment by arihantparsoya** [comment]
tests are passing?

**Pull request merged**